### PR TITLE
Added name and description as localizable.

### PR DESCRIPTION
**Pull request by odie2**
Added name and description as localizable.

### CODE REVIEW

File: lib/ManualImageCropEditorWindow.php [hunk 1]
```diff
@@ -94,6 +94,15 @@ public function renderWindow() {
 				$src_file = str_replace($uploadsDir['baseurl'], $uploadsDir['basedir'], $src_file_url[0]);
 				$sizes = getimagesize($src_file);
 
+				$original[0] = $sizes[0];
+				$original[1] = $sizes[1];
+
+				if ($width > $sizes[0]) {
```
Comment (tomaszsita) [inline lib/ManualImageCropEditorWindow.php]: Hi Odie,
thanks for your PR.
Can you explain what's going on here?
(moze byc po polsku :) )

File: lib/ManualImageCrop.php [hunk 2]
```diff
@@ -207,6 +207,23 @@ public function cropImage() {
 		$src_w = max(0, $_POST['select']['w']) * $_POST['previewScale'];
 		$src_h = max(0, $_POST['select']['h']) * $_POST['previewScale'];
 		
+		$size = wp_get_image_editor( $src_file )->get_size();
+		
+		$is_higher = ( $dst_h > $size["height"] );
```
Comment (tomaszsita) [inline lib/ManualImageCrop.php]: Same in here, does it fix any problem you were having?

File: lib/ManualImageCropEditorWindow.php (hunk 1 above)
Comment (odie2) [inline lib/ManualImageCropEditorWindow.php]: $original is just for echo in _e('Original picture dimensions:','microp') on "summary".

if ($width > $sizes[0]) { opens scaling to wider image only if needed.

File: lib/ManualImageCrop.php (hunk 2 above)
Comment (odie2) [inline lib/ManualImageCrop.php]: $size gets original uploaded image dimensions so $is_higher and $is_wider (booleans) are variable condition to check what scale should be done in next steps.

**Pull request merged**